dbos/_dbos_config.py

```diff
@@ -187,17 +189,26 @@ def translate_dbos_config_to_config_file(config: DBOSConfig) -> ConfigFile:
         ]
 
     # Telemetry config
-    telemetry = {}
+    telemetry: TelemetryConfig = {
+        "OTLPExporter": {"tracesEndpoint": [], "logsEndpoint": []}
+    }
+    assert telemetry["OTLPExporter"] is not None, "This is to make mypy happy"
```

Comment:
```suggestion
    assert telemetry["OTLPExporter"] is not None
```

Reply:
Since we change to use `List`, maybe they no longer need to be marked as `Optional`? This way, mypy wouldn't complain as well.